templates/settings.php

```diff
@@ -34,7 +34,7 @@
 				/>
 				<label for="av_max_file_size" class="a-left"><?php p($l->t('bytes'))?></label>
 			</p>
-			<p class="infected_action"><label for="av_infected_action"><?php p($l->t('Action for infected files found while scanning'));?></label>
+			<p class="infected_action"><label for="av_infected_action"><?php p($l->t('When infected files are found by cron job'));?></label>
```

Comment:
"When infected files were found during a background scan" ?

Or is it too long ?